src/test/java/eu/dissco/core/digitalspecimenprocessor/utils/TestUtils.java

```diff
@@ -32,6 +37,9 @@ public class TestUtils {
   public static JsonNode ORIGINAL_DATA = generateSpecimenOriginalData();
   public static String DWCA_ID = null;
 
+  public static final byte[] LOCAL_OBJECT_ID = "0x12:RMNH.QW99".getBytes(StandardCharsets.UTF_8);
```

Comment:
This doesn't look like an actual object id. Might be nice to take something a bit closer to the truth like.
`002b51e5-b8e1-4b2b-a841-86c34dca9ef6:040ck2b86`